clis/teliod/src/daemon.rs

```diff
@@ -25,19 +31,31 @@ pub enum TelioTaskCmd {
     UpdateMeshmap(MeshMap),
     // Get telio status
     GetStatus(oneshot::Sender<TelioStatusReport>),
+    // Enable or disable meshnet
+    SetMeshnet(bool, oneshot::Sender<Result<(), TeliodError>>),
+    // Configure the system interface
+    SetIp(oneshot::Sender<Result<(), TeliodError>>),
     // Break the receive loop to quit the daemon and exit gracefully
     Quit,
 }
 
 const EMPTY_TOKEN: &str = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
 
+#[derive(Debug, Default)]
+pub struct TelioTaskStates {
+    interface_ip_address: Option<IpAddr>,
+    meshnet_is_enabled: bool,
+}
```

Comment:
do we really need to keep track of this?

we can get the ip address from the meshmap, and check if telio is running through `telio.is_running()`

Reply:
`telio.is_running()` does not return the status of meshnet, but whole libtelio runtime.

Yes, in that case we would need to keep in memory the whole `meshmap` but since we only need the `IpAddr` I thought it would be an overkill.

Reply:
makes sense.